__tests__/app.test.js

```diff
@@ -27,18 +31,126 @@ describe("/api/categories", () => {
+        category: "euro game",
+        created_at: "2021-01-18T10:00:20.514Z",
+        votes: 1,
+      };
+      return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review).toEqual(expected);
+        });
+    });
+    test("non-existing review id results in status 404 - msg 'review not found", () => {
+      return request(app)
+        .get("/api/reviews/1000")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toEqual("review not found");
+        });
+    });
+    test("non-numeric review id results in status 400 - msg 'bad request", () => {
+      return request(app)
+        .get("/api/reviews/review1")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toEqual("bad request");
+        });
+    });
+  });
+  describe("PATCH", () => {
+    const update = {
+      inc_votes: 4,
+    };
+    test("existing review id results in status 200 - responds with the updated correct review object", () => {
+      const expected = {
+        title: "Agricola",
+        designer: "Uwe Rosenberg",
+        owner: "mallionaire",
+        review_img_url:
+          "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+        review_body: "Farmyard fun!",
+        review_id: 1,
+        category: "euro game",
+        created_at: "2021-01-18T10:00:20.514Z",
+        votes: 5,
+      };
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(update)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review).toEqual(expected);
+        });
+    });
+    test("non-existing review id results in status 404 - msg 'review not found", () => {
+      return request(app)
+        .patch("/api/reviews/1000")
+        .send(update)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toEqual("review not found");
+        });
+    });
+    test("non-numeric review id results in status 400 - msg 'bad request", () => {
+      return request(app)
+        .patch("/api/reviews/review1")
+        .send(update)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toEqual("bad request");
+        });
+    });
+    test("non-numeric vote results in status 400 - msg 'bad request", () => {
+      const wrongUpdate1 = {
+        inc_votes: "invalid",
+      };
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(wrongUpdate1)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toEqual("bad request");
+        });
+    });
+    test("invalid request key results in status 400 - msg 'bad request", () => {
+      const wrongUpdate2 = {
+        invalid: 4,
+      };
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(wrongUpdate2)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toEqual("bad request");
         });
```

Comment:
As we may add additional functionality to our patch requests later, I'd just expect this request to be ignored and return a 200 with the article unchanged.